.config/requirements-lock.txt

```diff
@@ -1,10 +1,10 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
This change is unnecessary and brakes linting.

Reply:
@ziegenberg That was autogenerated, I'm using 3.10. 

I've got a wip to add docs for how to update deps, tbh I found it a little confusing. Can you verify that this is the correct procedure?


### Updating Dependencies

Dependencies need to be updated by hand in:
  - `.config/requirements.in`
  - `.pre-commit-config.yaml` (2 places)

Afterwards, you will need to generate changes to `requirements.lock.txt`
and `requirement.txt`, by running the commands listed at the top of those files.

Please note that CI will attempt to regenerate those changes, and if there is any diff, CI will fail.

Reply:
You should run `tox -e lint` before pushing. That would save you some CI failures.